client/src/components/Comments/CommentBox.tsx

```diff
@@ -1,6 +1,9 @@
 import React, { Component } from 'react';
+import { SubmitVoid } from '../../types/dom-events-types';
 
-class CommentBox extends Component {
+type Props = SubmitVoid;
+
+class CommentBox extends Component<Props> {
```

Comment:
if this component does not have a state we can change it to dump or stateless functional component. If it has state we need to define type for state